### PR TITLE
Add dockerfile

### PR DESCRIPTION
**Pull request by tomahawk28**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+FROM alpine:latest
```
Comment (rossf7) [inline Dockerfile]: We don't use latest tags because we want building our images to be reproducible. We should use the `3.8` tag.

**Comment by tomahawk28** [comment]
I created quay repo but I'm not sure whether it should be private or not 
https://quay.io/repository/giantswarm/app-operator

File: README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+[![CircleCI](https://circleci.com/gh/giantswarm/app-operator.svg?style=shield)](https://circleci.com/gh/giantswarm/chart-operator) [![Docker Repository on Quay](https://quay.io/repository/giantswarm/chart-operator/status "Docker Repository on Quay")](https://quay.io/repository/giantswarm/app-operator)
```
Comment (indyfree) [inline README.md]: The badges partly reference `chart-operator` instead of app-operator

**Comment by tomahawk28** [comment]
I'm still getting this error from quay. 
```A build step failed: ADD failed: stat /var/lib/docker/tmp/docker-builder156319217/app-operator: no such file or directory```

I thought `architect` do that job but looks like it didn't add `app-operator`

**Comment by rossf7** [comment]
The Quay repo is still configured as an automated build. We don't use those because `architect` does our image builds.

Deleting the build trigger should fix it. In future you just need to create the Quay image as an empty repo.

screen shot 2018-12-17 at 12 42 05 pm

**Comment by tomahawk28** [comment]
it's now updated. https://quay.io/repository/giantswarm/app-operator?tab=tags 
Can I merge it?

**Comment by tomahawk28** [comment]
🤦‍♂️ Sorry, but thanks for updating the setting

**Pull request merged**